### DISCUSSION TITLE
Velero CSI Plugin functionality

### DISCUSSION THREAD

**Discussion by DSO-TB** [vmware-tanzu/velero#5132] [General discussions]
I have a question about some Velero functionality if anyone could help me understand better! I'm trying to use the CSI Plugin and from my understanding the CSI snapshotter should be able to backup the volumes without the need of EBS snapshots however Velero is still creating EBS snapshots anyway.  I think I might be missing some understanding of the functionality. 

My use case is that I am trying to perform cross account backups without using restic as it is only considered to be beta quality.

**Comment by sseago** [comment]
The CSI snapshotter relies on the underlying CSI driver, so the EBS snapshots aren't taken by Velero, but they *are* taken by the EBS CSI driver.

What you're trying to do will require the (not yet implemented) Data Mover functionality which will work with the CSI plugin to create out-of-cluster backup for CSI snapshots.
